TU20Bot/Configuration/Config.cs

```diff
@@ -101,16 +103,18 @@ public class Config {
             var environmentToken = Environment.GetEnvironmentVariable(tokenVariableName);
             var environmentMongo = Environment.GetEnvironmentVariable(mongoVariableName);
             var environmentDatabase = Environment.GetEnvironmentVariable(databaseVariableName);
-
-            if (string.IsNullOrEmpty(environmentToken))
+            var environmentJwtSecret = Environment.GetEnvironmentVariable(jwtSecretVariableName);
+            
+            if (string.IsNullOrEmpty(environmentToken) || string.IsNullOrEmpty(environmentJwtSecret))
```

Comment:
I like to think that the bot will still be able to operate without the JWT Secret. Can we make it so it works like `mongoUrl` and `databaseName`?

Reply:
> so that the server will not run if the secret is not available.
Hmm... I was thinking that we should allow the server *to* run if the secret is not available, as it is not a requirement for developers who are just debugging their own discord commands. Users who aren't making changes to the Web API won't be using the JWT, and I believe its unfair to force them to set one as it might complicate setup.

Reply:
Mongo is also used by a significant (well maybe) part of the project and yet we keep it optional. It would be nice to have an option to disable authentication, entirely during a debug environment.

Future developers will thank you when they don't have to call the `/login` endpoint for every hour of every debug session :)